db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Retirement/StateMachines/Methods.class/__methods__/check_pre_retirement.yaml

```diff
@@ -3,7 +3,7 @@ object_type: method
 version: 1.0
 object:
   attributes:
-    name: openstack_check_pre_retirement
+    name: check_pre_retirement
```

Comment:
@mkanoor 
I like using multi-messaging and would like to utilize that more going forward, but wanted a consistent approach with that used in infrastructure. I'm concerned about having to modify the class schema as you mentioned. Could we consider that approach for the next release?